test/index.ts

```diff
@@ -184,6 +184,12 @@ async.each(
 
       before(() => {
         Object.assign(fakeEntity, fakeEntityInit);
+        const buildEntityProtoModule = proxyquire(
+          '../src/utils/entity/buildEntityProto.js',
+          {
+            '../../entity.js': {entity: fakeEntity},
+          }
+        );
```

Comment:
These tests replace `entity` with `fakeEntity` using `'./entity.js': {entity: fakeEntity},` in the `src/index.ts` file. However, since the code that builds the entity proto used in these tests is no longer in `src/index.ts`, but is in `buildEntityProto.js` instead then the entity in `buildEntityProto.js` needs to be mocked out as well.